Formula/thors-sql.rb

```diff
@@ -0,0 +1,58 @@
+class ThorsSql < Formula
+  desc "SQL library for C++14"
+  homepage "https://github.com/Loki-Astari/ThorsSQL"
+  url "https://github.com/Loki-Astari/ThorsSQL.git",
+      :tag => "1.5.6",
+      :revision => "2da4ab817a4f45f0d3f8a7bf7ad0fc455031f0f6"
+
+  depends_on "mysql" => :build
+  depends_on "openssl"
+
+  needs :cxx14
+
+  def install
+    ENV["COV"] = "gcov"
+    system "mysql.server", "start"
+    system "init/testdb"
+    system "./configure", "--disable-vera",
+                          "--prefix=#{prefix}"
+    system "make", "-j", "1"
```

Comment:
You can use `ENV.deparallelize` here instead.

Reply:
Will update to do that.